Tools/autotest/common.py

```diff
@@ -1028,11 +1028,11 @@ def should_fetch_all_for_parameter_change(param_name):
             return True
         return False
 
-    def set_parameter(self, name, value, add_to_context=True, epsilon=0.0002):
+    def set_parameter(self, name, value, add_to_context=True, epsilon=0.0002, retries=10):
         """Set parameters from vehicle."""
         self.progress("Setting %s to %f" % (name, value))
         old_value = self.get_parameter(name, retry=2)
-        for i in range(1, 10):
+        for i in range(1, retries+2):
```

Comment:
Why +2 ? This looks to be trying harder then it was before.

Reply:
This loop now goes all the way to 11.  Need I say more?